### ISSUE TITLE
java.lang.NullPointerException when create new Ui Component Grid

### ISSUE THREAD

**Issue by DmitryFurs** [magento/magento2-phpstorm-plugin#475]
**Describe the bug**

When I try to create a new **Ui Component Grid** I catch the following exception:

```
java.lang.NullPointerException
	at com.magento.idea.magento2plugin.actions.generation.dialog.NewUiComponentGridDialog.getCollectionOptions(NewUiComponentGridDialog.java:484)
	at com.magento.idea.magento2plugin.actions.generation.dialog.NewUiComponentGridDialog.createUIComponents(NewUiComponentGridDialog.java:455)
	at com.magento.idea.magento2plugin.actions.generation.dialog.NewUiComponentGridDialog.$$$setupUI$$$(NewUiComponentGridDialog.java)
	at com.magento.idea.magento2plugin.actions.generation.dialog.NewUiComponentGridDialog.<init>(NewUiComponentGridDialog.java:201)
	at com.magento.idea.magento2plugin.actions.generation.dialog.NewUiComponentGridDialog.open(NewUiComponentGridDialog.java:245)
	at com.magento.idea.magento2plugin.actions.generation.NewUiComponentGridAction.actionPerformed(NewUiComponentGridAction.java:56)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAware(ActionUtil.java:281)
	at com.intellij.openapi.actionSystem.impl.ActionMenuItem$ActionTransmitter.lambda$actionPerformed$0(ActionMenuItem.java:310)
	at com.intellij.openapi.wm.impl.FocusManagerImpl.runOnOwnContext(FocusManagerImpl.java:286)
	at com.intellij.openapi.wm.impl.IdeFocusManagerImpl.runOnOwnContext(IdeFocusManagerImpl.java:77)
	at com.intellij.openapi.actionSystem.impl.ActionMenuItem$ActionTransmitter.actionPerformed(ActionMenuItem.java:299)
	at java.desktop/javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:1967)
	at com.intellij.openapi.actionSystem.impl.ActionMenuItem.lambda$fireActionPerformed$0(ActionMenuItem.java:110)
	at com.intellij.openapi.application.TransactionGuardImpl.performUserActivity(TransactionGuardImpl.java:95)
	at com.intellij.openapi.actionSystem.impl.ActionMenuItem.fireActionPerformed(ActionMenuItem.java:110)
	at com.intellij.ui.plaf.beg.BegMenuItemUI.doClick(BegMenuItemUI.java:514)
	at com.intellij.ui.plaf.beg.BegMenuItemUI$MyMouseInputHandler.mouseReleased(BegMenuItemUI.java:544)
	at java.desktop/java.awt.Component.processMouseEvent(Component.java:6652)
	at java.desktop/javax.swing.JComponent.processMouseEvent(JComponent.java:3345)
	at java.desktop/java.awt.Component.processEvent(Component.java:6417)
	at java.desktop/java.awt.Container.processEvent(Container.java:2263)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5027)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2321)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4859)
	at java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4918)
	at java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4547)
	at java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4488)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2307)
	at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2780)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4859)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:778)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:95)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:751)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:749)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:748)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:976)
	at com.intellij.ide.IdeEventQueue.dispatchMouseEvent(IdeEventQueue.java:911)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:840)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:454)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:773)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$9(IdeEventQueue.java:453)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:822)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:507)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```

**To Reproduce**

1. Create new Module project in PhpStorm
2. Right click on project root
3. Create **Magento 2 Ui Component Grid**
4. See error in **IDE Fatal Errors** section

**Expected behavior**

The successful creation of the component. Absence of any errors in the IDE Fatal Errors section.

**Screenshots**

![image](https://user-images.githubusercontent.com/1055999/106894734-68bfe700-6700-11eb-9218-dc0800ec050d.png)

![image](https://user-images.githubusercontent.com/1055999/106894790-77a69980-6700-11eb-89ca-aa9b5b5afde2.png)

**Please complete the following information:**

 - OS: MacOS
 - PhpStormversion: 2020.3.2
 - Plugin Version: 3.0.4

**Additional context**

![image](https://user-images.githubusercontent.com/1055999/106894908-9f95fd00-6700-11eb-8625-83615500b3c4.png)

**Comment by VitaliyBoyko** [comment]
Hi @DmitryFurs 
The root of the problem seems to be the same as in #446 and should be fixed in the next release. Stay tuned.

**Comment by eduard13** [comment]
Fixed in [3.1.0](https://github.com/magento/magento2-phpstorm-plugin/releases/tag/3.1.0) release
